public/lib/loadSwaggerUI.js

```diff
@@ -16,6 +16,7 @@ $(function() {
 
   var accessToken;
   function loadSwaggerUi(config) {
+    var methodOrder = ['get', 'head', 'put', 'post', 'delete'];
```

Comment:
Well there is a recent issue asking for that - see #110.

I can live with the current implementation too, I think it will order "options" as the first entry (find returns -1).